uast/libuast/helpers.c

```diff
@@ -0,0 +1,161 @@
+#include "uast.h"
+#include "uast_go.h"
+
+#include <stdlib.h>
+
+NodeKind _uastKind(const Uast* ctx, NodeHandle node) {
```

Comment:
You are right, it's only because Go cannot express C const parameters natively.

But by C semantic these parameters should be const. So I think it makes sense to leave this boilerplate in place to have a clean interface on C side. These wrappers can be generated automatically anyway, so they won't be a problem to maintain.